packages/data-context/package.json

```diff
@@ -75,6 +75,7 @@
     "@types/sinon-chai": "3.2.8",
     "@types/stringify-object": "^3.0.0",
     "mocha": "7.0.1",
+    "react-docgen": "5.4.3",
```

Comment:
One thing that might be nice to do is see how much the binary increases if we add a dependency. Just something I thought of - I'm not sure what this uses under the hood, but I'm guessing it uses babel, which might bring in a bunch of extra node modules.

I think we could check it by building the binary, and just comparing the size vs the current production version.

Reply:
yeah... when I run `yarn`, it adds a ton of lines to `snapshot-meta.cache.json` which leads me to believe that this brings in a bunch of other modules (although I'm not sure exactly what that file does)